### ISSUE TITLE
Bug: GetProcess fails for process names containing whitespace on Linux

### ISSUE THREAD

**Issue by amuckstot30** [rdbo/libmem#266]
I've recently had some issues with libmem not displaying some processes on linux.
After some investigation I found that this seems to happen for processes that have spaces in their names.
I think the issue might be caused by this line in `get_stat_info`:
```rust
const char *scanstr = "%*s %*s %*s %d %*s %*s %*s %*s %*s %*s %*s %*s %*s %*s %*s %*s %*s %*s %*s %*s %*s %llu";
if (fscanf(stat_file, scanstr, &ppid, &starttime) == 0 || errno) {
	goto CLOSE_EXIT;
}
```
The second column in the `stat` file contains to process name in parentheses,
so if it contains any whitespace `scanf` will match the wrong column

**Comment by rdbo** [comment]
Good catch, I never encountered a process with whitespaces so I didn't think about that at the time

**Comment by rdbo** [comment]
Fixed by 2aac816f537cabe4e0e217fbb3c2ea0b5836291d
Feel free to reopen if it doesnt work
Thanks for reporting!
